examples/generic_connector_configs/README.md

```diff
@@ -1,30 +1,232 @@
 # Generic HTTP Connector Example Configurations
 
+## Table of Contents
```

Comment:
I think this should follow the format of the [README ToC](https://github.com/cyberark/secretless-broker/blob/master/README.md#table-of-contents). I don't think it needs the links to itself and the main title. I also think it shouldn't link to the examples and "how tos" in all of them. 

```suggestion
## Table of Contents
- [Sample Configurations](#sample-configurations)
   - [Elasticsearch API](#elasticsearch-api)
   - [Github API](#github-api)
   - [OAuth 2.0 API](#oauth-20-api)
   - [Slack Web API](#slack-web-api)
   - [Splunk API](#splunk-api)
- [Contributing](#contributing)
```